src/views/checkups/network/components/CheckupsNetworkActions.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { IoK8sApiCoreV1ConfigMap } from '@kubevirt-ui/kubevirt-api/kubernetes';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownPosition,
+  DropdownToggle,
+  KebabToggle,
+} from '@patternfly/react-core';
+
+import { deleteNetworkCheckup, rerunNetworkCheckup, STATUS_SUCCEEDED } from '../utils/utils';
+
+const CheckupsNetworkActions = ({
```

Comment:
im sure you're right about the English :)
I choose this way because of routing, as checkups are first, then network, then form, etc...
if there is something that is backwards I think its better change it like u mention some are netwrokcheckups 

wdyt?

Reply:
I don't think it's necessary to constrain component/file names to match the routing path since the way we approach each of those is different. These file names just read wrong to my brain.

Reply:
I understand what you are saying, and u have a point, but currently, I can see this is how we are using it in many places. It's in our guidelines (need to find the link) that is written it should be like that, VM list, vmi list, and templates are using this convention, so I'm guessing it is okay here

Reply:
I believe you, but don't remember that discussion. Let me know if you find the link.